typescript-conditionals/main.ts

```diff
@@ -0,0 +1,108 @@
+/* exported isUnderFive,
+            isEven,
+            startsWithJ,
+            isOldEnoughToDrink,
+            isOldEnoughToDrive,
+            isOldEnoughToDrinkAndDrive,
+            categorizeAcidity,
+            introduceWarnerBro,
+            recommendMovie
+ */
+function isUnderFive(number: number): boolean {
+  if (number < 5) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function isEven(number: number): boolean {
+  if (number % 2) {
```

Comment:
`%` is the remainder operator. So if `number % 2` is `true`, that technically means the number is odd, because there's a remainder. You need to do `number % 2 === 0` to make sure there's no remainder